lib/network/Images.js

```diff
@@ -31,8 +31,12 @@ class Images{
      * @return {Image} imageToLoadBrokenUrlOn   The image object
      */    
     _tryloadBrokenUrl (url, brokenUrl, imageToLoadBrokenUrlOn) {
-        //If any of the parameters aren't specified then exit the function because nothing constructive can be done
-        if (url === undefined || brokenUrl === undefined || imageToLoadBrokenUrlOn === undefined)  return;
+        //If these parameters aren't specified then exit the function because nothing constructive can be done
+        if (url === undefined || imageToLoadBrokenUrlOn === undefined)  return;
```

Comment:
Is there any path that would lead to a undefined url here? If the check is for debugging of a case that should never happen then maybe an internal error print/throw make sense ?